### PR TITLE
plotting only simulation

### LINKED ISSUE #260: implement way to plot only simulation

### PR DESCRIPTION
**Pull request by PaulJonasJost**
Closes #260

### CODE REVIEW

**Comment by plakrisenko** [comment]
Can you also add a test to test_visualization.py, covering this case?

File: petab/visualize/plot_data_and_simulation.py [hunk 1]
```diff
@@ -107,10 +107,19 @@ def plot_data_and_simulation(
...
+        if sim_data is None:
+            plot_sim = 0
+        else:
+            plot_sim = 1
+        # import from file in case experimental data is provided in file
+        if isinstance(exp_data, str):
+            exp_data = measurements.get_measurement_df(exp_data)
+    else:
+        plot_sim = 2
+        exp_data = sim_data.rename(columns={'simulation': 'measurement'})
```
Comment (dilpath) [inline petab/visualize/plot_data_and_simulation.py:120]:
```suggestion
        exp_data = sim_data.rename(columns={SIMULATION: MEASUREMENT})
```

**Comment by dilpath** [comment]
Might be nicer to rewrite this such that `plot_sim`  is a set containing `petab.C.PLOT_TYPE_DATA` and/or `petab.C.PLOT_TYPE_SIMULATION` and then replace e.g.
```python
if plot_sim == 0 or plot_sim == 1:
```
with
```python
if PLOT_TYPE_DATA in plot_types:  # renamed `plot_sim` to `plot_types`
```

**Comment by dweindl** [comment]
Closing this PR here. Please create a new one towards https://github.com/PEtab-dev/libpetab-python

**Pull request closed without merging**